layouts/_default/baseof.html

```diff
@@ -16,10 +16,26 @@
     {{ end }}
   >
     <div class="wrapper">
-      <aside class="wrapper__sidebar">
+      <aside
+        {{ if and (not .IsHome) (eq .Site.Params.hidesidebar true) }}
+          class="wrapper__sidebar--sidebarhidden"
```

Comment:
Now, declarations of `wrapper__sidebar` would not be applied anymore. In case it's unclear, the nested SCSS simply resolves to definite class names. This means `wrapper__sidebar--sidebarhidden` of course does not include declarations of parents it's nested in.

Furthermore, semantically with BEM it does not make sense building standalone modifier classes. Modifiers are expected to be extensions as they add _modifications_ to blocks: `wrapper__sidebar wrapper__sidebar--hidden`.